aclk/agent_cloud_link.c

```diff
@@ -1635,7 +1637,8 @@ int aclk_send_info_metadata()
     buffer_flush(local_buffer);
     local_buffer->contenttype = CT_APPLICATION_JSON;
 
-    aclk_create_header(local_buffer, "connect", msg_id);
+    // Time is the session start by definition
+    aclk_create_header(local_buffer, "connect", msg_id, aclk_session_sec, aclk_session_us);
```

Comment:
Just making sure -- This means the `timestamp` and `timestamp-offset-usec` will be the same as `aclk_session_sec` `aclk_session_us` if a health_reload is done.

Reply:
This is the on-connect payload, why is it sent on a health_reload?

Reply:
health reload maybe have caused changes in alarm configuration. It is also called after a label reload so things may have changed there as well

Reply:
I can only see one call to `aclk_send_info_metadata` from `aclk_send_metadata` to send the on-conect payload. How does it called in the other case?

Reply:
It is called from `health_reload`  it calls `aclk_alarm_reload`

Reply:
Ok, so it is not a call in the code. It queues an on connect command instead. That will be a problem, not sure how we will fix that.